jdaviz/configs/default/plugins/export/export.py

```diff
@@ -254,8 +256,27 @@ def _set_filename_default(self, *args, **kwargs):
         else:
             self.filename_default = ''
 
+        # Call to initially set the default filepath
+        if hasattr(self, 'viewer_format') and self.filename_default:
+            self._normalize_filename(filename=self.filename_default,
+                                     filetype=self.viewer_format.selected,
+                                     default_path=True)
+
     @observe('filename_value')
     def _is_filename_changed(self, event):
+        # Update the UI Filepath if relative or absolute paths are provided
+        # by user via self.filename_value
+        if '~' in self.filename_value and '/' in self.filename_value:
+            expanded_path = os.path.expanduser(self.filename_value)
+            abs_path = Path(expanded_path).resolve()
+            self.default_filepath = str(abs_path)
+        elif '/' in self.filename_value and \
+             ('.' in self.filename_value or '..' in self.filename_value):
+            abs_path = Path(self.filename_value).resolve()
+            self.default_filepath = str(abs_path)
+        elif '/' in self.filename_value:
+            self.default_filepath = str(self.filename_value)

```

Comment:
The file path separator `/` is specific to unix machines. I don't expect this to work as expected on Windows. Is there any reason not to resolve every file path with the same `Path` methods?